docs/installation/setup_configuration.rst

```diff
@@ -0,0 +1,66 @@
+.. _installation_configuration_cli:
+
+==============================
+Open Klant configuration (CLI)
+==============================
+
+After deploying Open Klant, it needs to be configured to be fully functional.
+You can get the full command documentation with:
+
+.. code-block:: bash
+
+    $ src/manage.py setup_configuration --help
+
+.. warning:: This command is declarative - if configuration is manually changed after
+   running the command and you then run the exact same command again, the manual
+   changes will be reverted.
+
+Preparation
+===========
+
+The command executes the list of pluggable configuration steps, and each step
+has required specific variables, that should be prepared.
+Here is the description of all available configuration steps and the variables,
+used by each step.
+
+Token configuration
+----------------------
+
+Create a YAML configuration file with your settings:
```

Comment:
I added this on the line the comment was made on, all the previous documentation is set up quite generic (not mentioning either environment variables or `.yaml` files). As you mentioned earlier, this is like you mentioned subject to change (because of the documentation generation that will be updated for django-setup-configuration).